File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/ReportSubmitForm.java
package com.sdkwork.app.model;

public class ReportSubmitForm {
    private String type;
    private String targetId;
    private String content;
    private String attachmentUrl;
    private String screenshotUrl;

    public String getType() {
        return this.type;
    }
    
    public void setType(String type) {
        this.type = type;
    }

    public String getTargetId() {
        return this.targetId;
    }
    
    public void setTargetId(String targetId) {
        this.targetId = targetId;
    }

    public String getContent() {
        return this.content;
    }
    
    public void setContent(String content) {
        this.content = content;
    }

    public String getAttachmentUrl() {
        return this.attachmentUrl;
    }
    
    public void setAttachmentUrl(String attachmentUrl) {
        this.attachmentUrl = attachmentUrl;
    }

    public String getScreenshotUrl() {
        return this.screenshotUrl;
    }
    
    public void setScreenshotUrl(String screenshotUrl) {
        this.screenshotUrl = screenshotUrl;
    }
}

File: sdkwork-app-sdk-typescript/src/types/report-submit-form.ts
/** 举报提交表单 */
export interface ReportSubmitForm {
  /** 举报类型 */
  type: string;
  /** 举报对象ID */
  targetId: string;
  /** 举报内容 */
  content: string;
  /** 附件URL */
  attachmentUrl?: string;
  /** 相关截图 */
  screenshotUrl?: string;
}
